### ISSUE TITLE
Losing selected value on focus

### ISSUE THREAD

**Issue by anoviel** [robrogers3/vue-single-select#8]
Hi i just found some unexpected behaviour:
After i selected a value from the dropdown my Vue DevTool shows the correct object in the value property.
clicking into the input immedantily deletes this value.
selecting a new value works fine.
this causes my other dependent components to do an unnessessary reload because the model changed to null while performing described situation.

i think it would be nice to change the value just if a new one is selected 
or reset to null if you press enter in the empty input or pressing escape

any thoughts about that?

Greetings,
Jan Radtke

**Comment by robrogers3** [comment]
Yeah, when I designed this I considered what should happen if you click back into the field after selecting a value. On a normal select, you are right nothing happens. 

What should probably happen is only mutate the currently selected value if the user changes the selection.

I'll look into this in the next couple days.
